migrate/v0_15/migrate.md

```diff
@@ -0,0 +1,91 @@
+# kava-8 Upgrade Instructions
+
+## Software Version and Key Dates
+
+- We will be upgrading from chain-id "kava-7" to chain-id "kava-8".
+- The version of Kava for kava-8 is [TODO: version]
+- Kava-7 validators should prepare to shutdown their nodes August 30th, 2021 at 13:00 UTC by setting `--halt-time` to `1630328400`
```

Comment:
For the kava-8 upgrade, I would like to use an official software upgrade proposal that specifies a block _height_ rather than block time. This way, chain will automatically shut down at the specified height and there's no ambiguity about the last block. 

Update guide to have a place holder for final block height and then say that it will be at approximately 13:00 UTC.